enterprise/api_client/enterprise.py

```diff
@@ -44,12 +46,16 @@ def get_content_metadata(self, enterprise_customer, enterprise_catalogs=None):
         content_metadata = OrderedDict()
         enterprise_customer_catalogs = enterprise_catalogs or enterprise_customer.enterprise_customer_catalogs.all()
         for enterprise_customer_catalog in enterprise_customer_catalogs:
-            response = self._load_data(
-                self.ENTERPRISE_CUSTOMER_CATALOGS_ENDPOINT,
-                resource_id=str(enterprise_customer_catalog.uuid),
-                traverse_pagination=True,
-                querystring={'page_size': 1000},
-            )
+            if should_use_enterprise_catalog_api():
```

Comment:
i don't think the switch logic should live here, but wherever we are calling this function. this client should have no knowledge of any other client.

Reply:
👍